src/Documentation/glossary.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'Glossary',
+  desc:
+    'This guide is aimed to familiarize the users with definitions to ' +
+    'relevant DVC concepts and terminologies which are frequently used.',
+  contents: [
+    {
+      name: 'Workspace directory',
+      match: ['workspace'],
+      desc:
+        'Also abbreviated as workspace, it is the root directory of a ' +
```

Comment:
It not necessarily the root directory. It all the files that are visible to the user in the project directory. I would check if there is a good explanation for Git.

Reply:
Agree. The name can just be `Workspace` as it refers to the entire DVC project or repo from what I understand. So both these terms should be mentioned as synonyms. Remember that the glossary helps us either disambiguate similar terms or confirm synonyms.

Reply:
Suggested desc: "The **workspace** or DVC **project** consists of all the files and directories under the location where DVC was initialized with `dvc init`. Its root directory will contain a `.dvc/` subdirectory with the special [DVC Files and Directories] (/doc/user-guide/dvc-files-and-directories)". Typically, the workspace also contains a Git **repository**, which intersects with the DVC project, but they're not exactly the same.

Reply:
@jorgeorpinel sounds a little bit complicated :) Too much information. The question is - what do we want user to understand there? Most likely that workspace == his/her project files/directories. Then we can mention the usually it a git repo, and .dvc, etc.

Reply:
How about:

The **workspace** contains all of your DVC **project** files and directories. It's typically also a Git **repository**. See also `dvc init` (should link to cmd ref).

Reply:
Yep, looks almost perfect!

I would probably do `DVC project` -> `project`. 

 ... files, directories and DVC-files

link to init + a link to DVC Files